### ISSUE TITLE
Handle class fields

### ISSUE THREAD

**Issue by amadare42** [RafaelSalguero/CSharp2TS#10]
Extension should handle fields conversions.

**Comment by RafaelSalguero** [comment]
Maybe we should put this optional with disabled by default. Do class fields are serialized by default by Newtonsoft.Json?

**Comment by amadare42** [comment]
Yes: https://dotnetfiddle.net/T4le5X#
Therefore, I don't think it should be disabled by default because it is actually sent in DTO by default. Anyway, since extension generates invalid typescript code when there are any fields, it should be fixed to handle this case in one way or another.
For example from code 
```
public class Foo {
    private int Bar = 42;
    private int Bar;
}
```
this will be generated:
```
export interface Foo  {
    private int Bar = 42;
    private int Baz;
}
```

**Comment by RafaelSalguero** [comment]
I agree, marking this as a bug since generates invalid code

**Comment by RafaelSalguero** [comment]
Closed with commit c07489aa9006c66059a9111173953e10232759f7 and version 0.0.22
